src/h5web/providers/Provider.tsx

```diff
@@ -15,11 +12,6 @@ interface Props {
 function Provider(props: Props): ReactElement {
   const { api, children } = props;
 
-  // Wait until metadata is fetched before rendering app
-  const { value: metadata, error } = useAsync(async () => {
-    return api.getMetadata();
-  }, [api]);
-
```

Comment:
We no longer fetch the whole metadata tree, so the provider no longer blocks the rendering of the app.